monai/bundle/workflows.py

```diff
@@ -307,15 +308,17 @@ def __init__(
         super().__init__(workflow_type=workflow_type, meta_file=meta_file, properties_path=properties_path)
         self.config_root_path = config_root_path
         logging_file = str(self.config_root_path / "logging.conf") if logging_file is None else logging_file
-        if logging_file is not None:
+        if not logging_file:
+            logger.warn(f"Logging file is set to {logging_file}, skipping logging.")
+        elif logging_file is not None:
```

Comment:
I think After L310 the logging file cannot be None anymore.

So we could just put else here?